cmd/execution_config.go

```diff
@@ -85,13 +79,7 @@ func (exeConf *ExecutionConfig) SetupFlags(flags *pflag.FlagSet) {
 		"threshold for logging script execution")
 	flags.DurationVar(&exeConf.computationConfig.ScriptExecutionTimeLimit, "script-execution-time-limit", computation.DefaultScriptExecutionTimeLimit,
 		"script execution time limit")
-	flags.StringVar(&exeConf.preferredExeNodeIDStr, "preferred-exe-node-id", "", "node ID for preferred execution node used for state sync")
 	flags.UintVar(&exeConf.transactionResultsCacheSize, "transaction-results-cache-size", 10000, "number of transaction results to be cached")
-	flags.BoolVar(&exeConf.syncByBlocks, "sync-by-blocks", true, "deprecated, sync by blocks instead of execution state deltas")
-	flags.BoolVar(&exeConf.syncFast, "sync-fast", false, "fast sync allows execution node to skip fetching collection during state syncing,"+
-		" and rely on state syncing to catch up")
-	flags.IntVar(&exeConf.syncThreshold, "sync-threshold", 100,
-		"the maximum number of sealed and unexecuted blocks before triggering state syncing")
```

Comment:
When this PR is merged, these flags will be removed and it might cause interruptions on ansible, terraform or other deployment toolkits for execution nodes. 
cc: @sjonpaulbrown